app/css/openmrs-addonmanager.css

```diff
@@ -70,3 +70,125 @@ header .logo img {
     align-self: center;
 }
 
+.nav-header li {
+    color: #eeeeee;
+    font-size: 15px;
+    padding-right: 20px;
+    display: inline;
+    list-style: none;
+}
+
+header>ul,
+a {
+    color: #eeeeee;
+    padding-right: 2px;
+    margin-top: 15px;
+}
+
+.glyphicon {
+    color: #eeeeee;
+    font-size: 12px;
+}
+
+li>a:hover {
+    color: #eeeeee;
+    cursor: pointer;
+    text-decoration: none;
+    border-bottom: 1px solid gold;
+}
+
+li>a:focus {
+    color: gold;
+}
+.table thead {
+    position: -webkit-sticky;
+    position: -moz-sticky;
+    position: -ms-sticky;
+    position: -o-sticky;
+    position: sticky;
+    top: 0;
+    background-color: #FFF;
+}
+
+a:hover {
+    text-decoration: none;
+}
+
+.dropdown-large {
+    position: static !important;
+}
+
+.dropdown-menu-large {
+    margin-left: 16px;
+    margin-right: 16px;
+    padding: 20px 0px;
+}
+
+.dropdown-menu-large>li>a:hover {
+    background-color: transparent;
+    margin-right: 0px;
+    border-bottom: 0px;
+    color: purple;
+}
+
+.MainHomePage h2{
+    font-size: 1.3em;
+    color: #009384;
+    margin-left: 5px;
+}
+
+.HomePageBody{
```

Comment:
For some, you start with lower case, for others, upper. Do you do this intentionally? 😊

Reply:
Okay, thank you for that. Consistency is important. I am changing this now.